.github/workflows/health_checks.yml

```diff
@@ -16,7 +16,7 @@ jobs:
         # Windows install must happen on the same worker size as subsequent jobs.
         # Larger workers use different drive (C: instead of D:) to check out project and NPM installation
         # creates file system links that include drive letter.
-        os: [ubuntu-latest, macos-latest, amplify-backend_windows-latest_8-core]
+        os: [ubuntu-latest, macos-latest, windows-latest]
```

Comment:
if this works, could you add more content to the comment above?
perhaps explaining that going back and forth between standard and custom agents require full install cache invalidation?